spec/features/review/user_updates_their_review_spec.rb

```diff
@@ -0,0 +1,69 @@
+require 'rails_helper'
+
+feature 'user updates their review', %{
+  As an authenticated user
+  I want to be able to changed my review
+  To reflect any new feelings I might have about a restaurant
+} do
+
+  # Acceptance Criteria
+  # [x] I must be signed in
+  # [x] If I'm not signed in, I should be prompted to do so
+  # [x] I can only edit my own reviews
+  # [x] The Edit Review form is autofilled with my original review/rating
+  # [x] When I submit my changes, I can see them on the restaurant's detail page
+  # [x] I get a successfully updated message
+  # [x] When user not signed in, and tries to visit reviews/edit page, user is
+  # redirected to sign in page
+
+  let!(:user) { FactoryGirl.create(:user) }
+  let!(:restaurant) { FactoryGirl.create(:restaurant) }
+  let!(:review) { FactoryGirl.create(
```

Comment:
Avoid using {...} for multi-line blocks.<br>Block body expression is on the same line as the block start.